docs/setup/changing-the-colors.md

```diff
@@ -212,28 +212,28 @@ The following fields must be set for each toggle:
 
 [:octicons-file-code-24: Source][6] · :octicons-milestone-24: Default: _none_
 
-In order to automatically set the color palette to the user's system preference,
-a media query can be set as part of the `media` field next to the toggle
-definition in `mkdocs.yml`:
+To match a color palette to the user's system preference upon the user's first
+visit, provide a media query via the `media` field next to the `scheme` definition
+in `mkdocs.yml`:
 
 ``` yaml hl_lines="3 8"
 theme:
   palette:
-    - media: "(prefers-color-scheme: light)"
```

Comment:
I don't know whether I've not seen this in the first review, because you force-pushed (please don't do that, as we loose track of changes). 

Why did you change the order? First of all, now the wrong lines are highlighted, and second of all I prefer `media` to be at the top, because it's more clear that the following settings are tied to it.

Reply:
Yeah, this was new, because it dawned on me after discussing "match[ing]" above that the ordering seemed…_off_.

Thanks for reminding me about the highlighting! Should be fixed now.

My suggestion is to put `scheme` first, because it is the primary thing everything else is anchored. Both `media` and `toggle` are subordinate to `scheme` no? (Meaning neither makes sense without `scheme` being present, correct?)

Reply:
You can set any of the primary, accent or scheme values regardless of each other, so there’s no parent relationship with the scheme.

The current order is more intuitive because it read like “for light mode I set primary to ...”

Reply:
Sorry for the confusion. I have reverted this.

For what it's worth, I was suggesting they were semantically dependent (at least given what was present in the example), not syntactically. I realize that they are peers in the YAML schema. I only meant to highlight that `media` and `toggle` are effectively meaningless without at least one of `primary`, `accent`, or `scheme` (and only `scheme` was present in the example). Consider, for example, the following configuration:

```yaml
theme:
  palette:
    - media: "(prefers-color-scheme: light)"
      toggle:
        icon: material/toggle-switch-off-outline
        name: Switch to some mode
    - media: "(prefers-color-scheme: dark)"
      toggle:
        icon: material/toggle-switch
        name: Switch to some other mode
```

In any event, I backed this out per your request.